warp10/src/main/java/io/warp10/script/MemoryWarpScriptStack.java

```diff
@@ -609,7 +609,7 @@ public void exec(String line) throws WarpScriptException {
           // End execution on encountering a comment
           //
 
-          if (!inMultiline.get() && stmt.length() > 0 && (stmt.charAt(0) == '#' || (stmt.charAt(0) == '/' && stmt.length() >= 2 && stmt.charAt(1) == '/'))) {
+          if (!inMultiline.get() && !inComment.get() && stmt.length() > 0 && (stmt.charAt(0) == '#' || (stmt.charAt(0) == '/' && stmt.length() >= 2 && stmt.charAt(1) == '/'))) {
```

Comment:
intuitively I would have thought that inComment.get() was the costliest of all those conditions, if that is the case then it would be better to move it at the end of the conditional to degrade performance less.

The same could be thought of inMultiline.get()